.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish Gem
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+
+      - name: Release Gem
+        uses: discourse/publish-rubygems-action@b55d7b91b55e61752dc6cbc2972f8e16fe6c1a02
```

Comment:
Can we still use original repo https://github.com/cadwallion/publish-rubygems-action and not 3rd level fork

Reply:
No it's broken. At the very least there needs to be a `gem update bundler` in there. Relevant issue in the original repo - https://github.com/cadwallion/publish-rubygems-action/issues/10.